dynamic.js

```diff
@@ -47,4 +47,133 @@ function closecContainer() {
 icon.addEventListener('click', openContainer);
 closeBtn.addEventListener('click', closecContainer);
 const navLink = document.querySelectorAll('.nav-click');
-navLink.forEach((a) => a.addEventListener('click', closecContainer));
\ No newline at end of file
+navLink.forEach((a) => a.addEventListener('click', closecContainer));
+
+// pop up window info
+
```

Comment:
The pop-up window does not show buttons on small and medium screens. Please see the screenshots below and fix the design issues.
![medium](https://user-images.githubusercontent.com/35504149/132757883-233fd42b-04a4-4a36-a380-62c40803f666.png)
![small](https://user-images.githubusercontent.com/35504149/132757890-bffcbfe6-f498-472f-945e-94c669f8ffeb.png)

Reply:
but in my browser, It shows buttons 
![image](https://user-images.githubusercontent.com/43178495/132758940-5fdf146d-d794-4359-b86b-6563bfa4ffbe.png)
and here is medium screen
![image](https://user-images.githubusercontent.com/43178495/132759144-8a6e4090-7fcd-4349-ac7b-f3b7732cb6de.png)

Reply:
another screenshot without inspecting element for mobile version
![image](https://user-images.githubusercontent.com/43178495/132759407-0a969f3c-b3cd-49a2-8116-6998ccff8336.png)

and here is the medium one
![image](https://user-images.githubusercontent.com/43178495/132759688-2c8713bd-e4e1-46e2-b790-e18746701f87.png)

Reply:
It seems the problem in your browser